helm/prometheus-rules/templates/alerting-rules/dex.rules.yml

```diff
@@ -22,3 +22,27 @@ spec:
         severity: page
         team: rainbow
         topic: dex
+    - alert: DexSecretExpired
+      annotations:
+        description: '{{`Dex operator running on {{ $labels.installation }} failed to renew secret of {{ $labels.app_registration_name }} for {{ $labels.app_owner }} on provider {{ $labels.provider_type }}.`}}'
+        opsrecipe: dex-operator/
+      expr: min by(app_registration_name, app_owner, app_namespace, provider_name, provider_type, installation) (aggregation:dex_operator_idp_secret_expiry_time) - time() < 60*60*12
+      for: 30m
+      labels:
+        area: kaas
+        cancel_if_outside_working_hours: "true"
+        severity: page
+        team: rainbow
+        topic: dex
+    - alert: ManagementClusterDexAppMissing
+      annotations:
+        description: '{{`Dex operator did not register a dex-app in {{ $labels.app_namespace }} namespace on {{ $labels.installation }}.`}}'
```

Comment:
There's no `installation` label.
I prefer to not specify it. as it's generated in the Opsgenie alert anyway.

Reply:
makes sense!